### PR TITLE
proxy chores

### PR DESCRIPTION
**Pull request by Omarabdul3ziz**
### Description
- rename counters to stats in proxy/client
- remove unused lru cache
- update some missing swagger docs

### Related Issues
- https://github.com/threefoldtech/tfgrid-sdk-go/issues/335
- https://github.com/threefoldtech/tfgrid-sdk-go/issues/336
- https://github.com/threefoldtech/tfgrid-sdk-go/issues/341
- https://github.com/threefoldtech/tfgrid-sdk-go/issues/345

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstring

### CODE REVIEW

File: grid-proxy/internal/explorer/server.go [hunk 1]
```diff
@@ -171,7 +169,7 @@ func (a *App) getNodes(r *http.Request) (interface{}, mw.Response) {
 // @Param rented_by query int false "rented by twin id"
 // @Param available_for query int false "available for twin id"
 // @Param farm_ids query string false "List of farms separated by comma to fetch nodes from (e.g. '1,2,3')"
-// @Param certification_type query string false "certificate type NotCertified, Silver or Gold" Enums(NotCertified, Silver, Gold)
+// @Param certification_type query string false "certificate type" Enums(Certified, DIY)
```
Comment (AbdelrahmanElawady) [inline grid-proxy/internal/explorer/server.go:173]: I think according to our [docs](https://manual.grid.tf/faq/faq.html#what-are-the-different-certifications-available-for-3node-servers-and-farms-what-are-the-gold-and-silver-certifications) it should be as it was, right?

Comment (Omarabdul3ziz) [reply grid-proxy/internal/explorer/server.go:173]: I couldn't verify this from the docs, is it applied for the farms only or for the farms and nodes? so I went to check the values in graphql scheme and it looks like we have `Gold` and `NotCertified` for farms and `Diy` and `Certified` for the nodes.
@xmonader  can we verifty this?

**Review by muhamadazmy — APPROVED** [review]

File: grid-proxy/internal/explorer/server.go (hunk 1 above)
Comment (xmonader) [reply grid-proxy/internal/explorer/server.go:173]: @Mik-TF are you aware of any changes there?

Comment (mik-tf) [reply grid-proxy/internal/explorer/server.go:173]: From what I know there aren't any changes. What @Omarabdul3ziz is saying with gold and notcertified for farms, and diy and certified for the nodes, should still be effective.

**Review by AbdelrahmanElawady — APPROVED** [review]

**Pull request merged**